<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgClientes</class>
 <widget class="QWidget" name="wdgClientes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>376</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Clientes</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblBusqueda">
         <property name="text">
          <string>Buscar Cliente</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtBusqueda"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tblClientes">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <column>
        <property name="text">
         <string>Id Cliente</string>
        </property>
        <property name="textAlignment">
         <set>AlignRight|AlignVCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nombre</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Apellidos</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>NIF</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Teléfono</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Correo electrónico</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Dirección</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ciudad</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Código Postal</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>País</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblFacturas">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Facturas</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tblFacturas">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Id Factura</string>
        </property>
        <property name="textAlignment">
         <set>AlignRight|AlignVCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Fecha</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Cliente</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Artículos</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Total venta</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Pagado</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionFacturaCliente">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/factura.png</normaloff>:/gnuoptics/factura.png</iconset>
   </property>
   <property name="text">
    <string>Factura nueva</string>
   </property>
  </action>
  <action name="actionFacturaClienteGenerico">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/factura.png</normaloff>:/gnuoptics/factura.png</iconset>
   </property>
   <property name="text">
    <string>Factura a cliente genérico</string>
   </property>
  </action>
  <action name="actionFacturaModificar">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/document-edit.png</normaloff>:/gnuoptics/document-edit.png</iconset>
   </property>
   <property name="text">
    <string>Modificar factura</string>
   </property>
  </action>
  <action name="actionFacturaBorrar">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/list-remove.png</normaloff>:/gnuoptics/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Borrar factura</string>
   </property>
  </action>
  <action name="actionClienteBorrar">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/list-remove.png</normaloff>:/gnuoptics/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Borrar cliente</string>
   </property>
   <property name="toolTip">
    <string>Borrar cliente</string>
   </property>
  </action>
  <action name="actionClienteModificar">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/document-edit.png</normaloff>:/gnuoptics/document-edit.png</iconset>
   </property>
   <property name="text">
    <string>Modificar cliente</string>
   </property>
   <property name="toolTip">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/gnuoptics/keke.png&quot; /&gt;Modifica al cliente&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionClienteNuevo">
   <property name="icon">
    <iconset resource="../images/gnuoptics.qrc">
     <normaloff>:/gnuoptics/contact-new.png</normaloff>:/gnuoptics/contact-new.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo cliente</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../images/gnuoptics.qrc"/>
 </resources>
 <connections/>
</ui>
